metadata-ingestion/setup.py

```diff
@@ -751,6 +752,7 @@
             "mongodb",
             "slack",
             "mssql",
+            "mssql-odbc",
```

Comment:
should also set up an alias in the source_registry, if that doesn't exist already